bundle/compliance/cis_aws/rules/cis_1_10/rule.rego

```diff
@@ -0,0 +1,23 @@
+package compliance.cis_aws.rules.cis_1_10
+
+import data.compliance.lib.common
+import data.compliance.policy.aws_iam.data_adapter
+
+default rule_evaluation = false
+
+# Ensure multi-factor authentication (MFA) is enabled for all IAM users that have a console password.
+rule_evaluation {
+	count(data_adapter.iam_user.MFADevices) > 0
+	data_adapter.iam_user.HasLoggedIn
+}
```

Comment:
should be part of an audit function, no?

Reply:
> Ensure that for each user where the `Password age` column shows a password age, the `MFA` column shows `Virtual`, `U2F Security Key`, or `Hardware`

also, the audit says mentions `U2F Security Key` and `Hardware` as valid options. Looking at the audit from the CLI it also looks different

Reply:
Cloudbeat fetches all user's MFA devices, not only virtual. 
Do you think that "Virtual, U2F Security Key, or Hardware" does not cover all the devices?

Reply:
oh I missed this part my bad